tests/components/Dropdown.test.tsx

```diff
@@ -44,7 +44,7 @@ describe('Dropdown', () => {
     expect(mockedOnToggleFn).toBeCalledWith(true, '');
   });
 
-  it('Keyboard navigation properly update focus on the option and input text', () => {
+  it('takes arrowkey navigation properly and updates focus on the option and input text', () => {
```

Comment:
takes is a weird verb here, I'd probably say "handles" or something

instead of "updates focus" maybe "focuses"?

Reply:
changed!